doc/py_tutorials/py_feature2d/py_sift_intro/py_sift_intro.markdown

```diff
@@ -102,9 +102,8 @@ reasons. In that case, ratio of closest-distance to second-closest distance is t
 greater than 0.8, they are rejected. It eliminates around 90% of false matches while discards only
 5% correct matches, as per the paper.
 
-So this is a summary of SIFT algorithm. For more details and understanding, reading the original
-paper is highly recommended. Remember one thing, this algorithm is patented. So this algorithm is
-included in [the opencv contrib repo](https://github.com/opencv/opencv_contrib)
+This is a summary of SIFT algorithm. For more details and understanding, reading the original
+paper is highly recommended.
```

Comment:
I would suggest to keep a note about the patent expiration. Something like these.

```.diff
-Remember one thing, this algorithm is patented.
+The patent on SIFT expired in 2020
```

Reply:
Sure, I added the note to the "SIFT in OpenCV" section, instead. It seemed a little out of context in the step-by-step algorithm summary section.